libs/common-types/src/lib/con-profile/enums/redi-course.enum.ts

```diff
@@ -34,17 +30,24 @@ export enum RediCourse {
   MUNICH_INTRODUCTION_TO_COMPUTER_SCIENCE_UKR = 'MUNICH_INTRODUCTION_TO_COMPUTER_SCIENCE_UKR',
   MUNICH_PYTHON_INTERMEDIATE_ONLINE = 'MUNICH_PYTHON_INTERMEDIATE_ONLINE',
   MUNICH_PYTHON_INTERMEDIATE_HYBRID = 'MUNICH_PYTHON_INTERMEDIATE_HYBRID',
+  MUNICH_PYTHON_INTERMEDIATE_UKR = 'MUNICH_PYTHON_INTERMEDIATE_UKR',
   MUNICH_FRONT_END_DEVELOPMENT_HTML_CSS = 'MUNICH_FRONT_END_DEVELOPMENT_HTML_CSS',
   MUNICH_FRONT_END_DEVELOPMENT_JAVASCRIPT = 'MUNICH_FRONT_END_DEVELOPMENT_JAVASCRIPT',
-  MUNICH_UX_UI_DESIGN = 'MUNICH_UX_UI_DESIGN',
+  MUNICH_FRONT_END_DEVELOPMENT_REACT = 'MUNICH_FRONT_END_DEVELOPMENT_REACT',
+  MUNICH_UX_UI_DESIGN = 'MUNICH_UX_UI_DESIGN_BASICS',
```

Comment:
I think this should be ...
```suggestion
  MUNICH_UX_UI_DESIGN_BASICS = 'MUNICH_UX_UI_DESIGN_BASICS',
```
, right?

Reply:
Great, feel free to correct and we can get this merged & deployed